src/Shared/Microsoft.AspNetCore.Razor.Utilities.Shared/LanguageSupport/InterpolatedStringHandlerAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+// Copied from https://github.com/dotnet/runtime
+
+#if !NET6_0_OR_GREATER
+
+namespace System.Runtime.CompilerServices;
+
+/// <summary>Indicates the attributed type is to be used as an interpolated string handler.</summary>
+[AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct, AllowMultiple = false, Inherited = false)]
+public sealed class InterpolatedStringHandlerAttribute : Attribute
```

Comment:
Oh, hold on, these need to be internal.

Reply:
Oops! Good catch! Thank you *very* much for that one.